cg/constants/demultiplexing.py

```diff
@@ -37,6 +37,7 @@ class DemultiplexingDirsAndFiles(StrEnum):
     FLOW_CELLS_DIRECTORY_NAME: str = "flow_cells"
     DEMULTIPLEXED_RUNS_DIRECTORY_NAME: str = "demultiplexed_runs"
     OUTPUT_FILE_MANIFEST: str = "Manifest.tsv"
+    CUSTOM_OUTPUT_FILE_MANIFEST: str = "custom_manifest.tsv"
```

Comment:
What to name this?

Reply:
DIR_FILE_MANIFEST?

Reply:
I want it to be clear that this is a file we create ourselves. Maybe `CG_FILE_MANIFEST: str = file_manifest.tsv`?

Reply:
Hard one - I do not have a better suggestion. Let's go with this 👍